packages/openapi-generator/src/parser/api.ts

```diff
@@ -22,8 +23,13 @@ export function parseApis(
   return Object.entries(operationsByApis).map(
     ([name, operations]: [string, OperationInfo[]]) => ({
       name,
-      operations: ensureUniqueOperationIds(
-        nameOperations(operations)
+      operations: ensureUniqueNames(
+        nameOperations(operations),
```

Comment:
Question: What would happen if the operations have names which are protected keywords in typescipt? I did not see a check in the `ensureUniqueNames` and `nameOperations` should one add the check in the `ensureUniqueNames` in the `UniqueNameFinder`?